cmd_rank.lua

```diff
@@ -0,0 +1,59 @@
+function HandleRankCommand(Split, Player)
+	local Response
+	local PlayerName = Split[2]
+	local NewRank = Split[3]
+	
+	local InformLoadRank = function(OtherPlayer)
+		if PlayerName == OtherPlayer:GetName() then
+			local AuthorName = "the server console"
+			
+			if Player then
+				AuthorName = "\"" .. Player:GetName() .. "\""
```

Comment:
Do we need the quotation marks?

Reply:
I personally prefer them, especially if the player has a username that blends in with the sentence.

Reply:
I don't really mind, but we should be consistent one way or the other. I see that in your other cleanup that we have the quotation marks so they're fine.